src/Features/Protocol/LanguageServices/Lsp/DocumentSymbol.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+// Straight copy until new Microsoft.VisualStudio.LanguageServer.Protocol nuget is published with this type in
+namespace Microsoft.VisualStudio.LanguageServer.Protocol
+{
+    /// <summary>
+    /// Represents programming constructs like variables, classes, interfaces etc. that appear in a document. Document symbols can be
+    /// hierarchical and they have two ranges: one that encloses its definition and one that points to its most interesting range,
+    /// e.g. the range of an identifier.
+    /// </summary>
+    [DataContract]
+    public class DocumentSymbol
```

Comment:
>public [](start = 4, length = 6)

Can we make these internal?

Reply:
These will be removed shortly, they're already added to Microsoft.VisualStudio.LanguageServer.Protocol , but waiting on a couple more changes before requesting them to build a new version with them in it.

Reply:
I see. OK.

---
In reply to: [274066441](https://github.com/dotnet/roslyn/pull/34794#discussion_r274066441) [](ancestors = 274066441)

Reply:
Looks like I need to make these public and add to API list until the new version is merged.